.github/workflows/ci_linting.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Linters
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  linters:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: cppcheck
+        run: |
+          source ./codebuild/bin/s2n_setup_env.sh
```

Comment:
This works for now, but if we need to do this every time, we should probably look into a more native way to do this, such as using an action like this[1] so that we can have this done as a separate step that preps for every following step rather than a command in every step.

[1] https://github.com/zweitag/github-actions

Reply:
I have short term concerns around maintainability, as the env script is used in all ci and by folks running this locally, but noted.

Reply:
Yeah, and I wouldn't want to duplicate efforts. Maybe some way to wrap one in the other or something?